src/Extensions/Logging/Internal/OmexLogger.cs

```diff
@@ -22,14 +22,19 @@ public OmexLogger(ILogEventSender logsEventSource, IExternalScopeProvider extern
 
 		public bool IsEnabled(LogLevel logLevel) => m_logsEventSender.IsEnabled(logLevel);
 
-		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception exception, Func<TState, Exception, string> formatter)
+		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
 		{
 			if (!IsEnabled(logLevel))
 			{
 				return;
 			}
 
 			string message = formatter(state, exception);
+			if (exception != null)
+			{
+				message = string.Concat(message, Environment.NewLine, exception); // we need to concatenate with exception since default formatter ignore it https://github.com/aspnet/Logging/issues/442
```

Comment:
```suggestion
				message = string.Concat(message, Environment.NewLine, exception); // We need to concatenate with exception since the default formatter ignores it https://github.com/aspnet/Logging/issues/442
```